main/inc/lib/social.lib.php

```diff
@@ -83,7 +83,33 @@ public static function get_relation_between_contacts($user_id, $user_friend)
 
             return $row['id'];
         } else {
-            return USER_UNKNOWN;
+            if(api_get_configuration_value('treat_superior_statuses_as_friends'))
+            {
+                $targetUserCoursesList = CourseManager::get_courses_list_by_user_id($user_id,true,false); //Gathering all the courses for the target user
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 153 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5726531)